.golangci.yml

```diff
@@ -29,5 +29,4 @@ linters-settings:
   dupl:
     threshold: 400
   goimports:
-    # Don't use 'github.com/kudobuilder/kudo', it'll result in unreliable output!
-    local-prefixes: github.com/kudobuilder
+    local-prefixes: github.com/kudobuilder/kudo
```

Comment:
what changed here? that we can do it now...

Reply:
No idea 😆
I had a lot of issues with that back when we introduced `golangci-lint`. Now I don't see them anymore and have been using it with full prefixes successfully in other (albeit smaller) projects -- [kitt](https://github.com/kudobuilder/kitt) and [test-tools](https://github.com/kudobuilder/test-tools).

Reply:
why don't we leave it `local-prefixes: github.com/kudobuilder`?  It seems to function and allows for a usable configuration across projects.

Reply:
Because that wouldn't just group `kudo` packages but packages from other projects in the `kudobuilder` org as well. For example in `pkg/engine/task/task_kudo_operator_test.go`, a `kuttl` import was grouped with `kudo` imports. This will be fixed with this change to `local-prefixes`.